tf-job-operator-chart/templates/dashboard.yaml

```diff
@@ -21,11 +21,14 @@ spec:
       labels:
         name: tf-job-dashboard
     spec:    
+{{- if .Values.rbac.install }}
+      serviceAccountName: tf-job-operator
```

Comment:
s/tf-job-operator/tf-job-dashboard ?

Reply:
👍  I will edit the chart to also create this separate service account for the dashboard and associated RBAC.